packages/fx-core/tests/plugins/resource/function/unit/expectedBicepFiles/functionConfigWithKeyVaultPlugin.result.bicep

```diff
@@ -11,12 +11,17 @@ param currentAppSettings object
 var functionAppName = split(provisionOutputs.functionOutput.value.functionAppResourceId, '/')[8]
 
 var m365ClientId = provisionParameters['m365ClientId']
+
 var m365ClientSecret = provisionOutputs.keyVaultOutput.value.m365ClientSecretReference
+
 var m365TenantId = provisionParameters['m365TenantId']
 var m365OauthAuthorityHost = provisionParameters['m365OauthAuthorityHost']
 var oauthAuthority = uri(m365OauthAuthorityHost, m365TenantId)
+
 var tabAppDomain = provisionOutputs.frontendHostingOutput.value.domain
-var tabAppEndpoint = provisionOutputs.frontendHostingOutput.value.endpoint 
+var tabAppEndpoint = provisionOutputs.frontendHostingOutput.value.endpoint
+
+
```

Comment:
we may also consider whether the generated bicep content looks good to our customers. For example, we'd better do not have 2 line breaks here.
Can we leverage some expressions in handlebars like `~` (the white space control) to remove the new line breaks which is caused by handlebars template change? What we want to achieve is: improve the readability of bicep handlebars template, and keep the generated content unchanged.

Reply:
will update in later pr.